core/modules/wbase/SendChannel.h

```diff
@@ -42,16 +42,17 @@ namespace wbase {
 /// accepts only one call to send bytes, unless the sendStream call is used.
 class SendChannel {
 public:
-    typedef util::VoidCallable<void> ReleaseFunc;
-    typedef std::shared_ptr<ReleaseFunc> ReleaseFuncPtr;
-    typedef long long Size;
+    using Ptr = std::shared_ptr<SendChannel>;
+    struct ReleaseFunc {
```

Comment:
Is there reason to prefer explicit struct to `util::VoidCallable`? It looks like it is doing the same thing. Should we think about using `std::function` for all of these?